### ISSUE TITLE
Can you tell me what the final loss of pretraining is?

### ISSUE THREAD

**Issue by lijiaoyang** [OpenMatch/MARVEL#3]
thxs

**Comment by whale-z** [comment]
Hello, during the pretraining phase, we employed image-caption contrastive training loss. The final training loss reached 0.9375, and the loss progression throughout the pretraining phase is illustrated in the figure below:

<img width="1078" alt="def909aaff16cd6533f4313c8e678355" src="https://github.com/user-attachments/assets/a32ecf33-5e30-49f5-8ef6-f7c6ddb10afb" />

If you still have questions, please contact us.
